pom.xml

```diff
@@ -69,10 +69,12 @@
       <artifactId>ionicons-api</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <scope>test</scope>
-    </dependency>
+    <groupId>org.mockito</groupId>
+    <artifactId>mockito-inline</artifactId>
+    <version>2.13.0</version>
+    
+    <scope>test</scope>
+</dependency>
```

Comment:
I see, I think we don't need to use mockito. In my case, I used Mockito because I did not have a jenkinsRule running, only later I added Jenkins rules. So keeping mocks does not make sense, I agree.

Reply:
I will revert the version.